src/com/jambit/onboarding2020/tbrpg/domain/Room/MobRoom.java

```diff
@@ -3,86 +3,185 @@
 import com.jambit.onboarding2020.tbrpg.core.GameInput;
 import com.jambit.onboarding2020.tbrpg.core.InvalidInputException;
 import com.jambit.onboarding2020.tbrpg.core.ItemGenerator;
+import com.jambit.onboarding2020.tbrpg.domain.Item.Item;
+import com.jambit.onboarding2020.tbrpg.domain.Item.Weapon;
 import com.jambit.onboarding2020.tbrpg.domain.Player.Enemy;
 import com.jambit.onboarding2020.tbrpg.domain.Player.EnemyDeadException;
 import com.jambit.onboarding2020.tbrpg.domain.Player.Player;
 import com.jambit.onboarding2020.tbrpg.domain.Player.PlayerDeadException;
 
 import java.io.InputStreamReader;
-import java.util.Random;
+import java.util.ArrayList;
+import java.util.Scanner;
 
 public class MobRoom extends AbstractRoom {
 
-   public void printRoomMessage() {
-      System.out.println("Um weiter zu kommen musst du einen Gegner bezwingen");
-   }
 
-   @Override
-   public void printWelcomeMessage() {
-      System.out.println("Du stehst einem mächtigen Gegner gegenüber.");
-   }
+    Player player = Player.getPlayerInstance();
+    Scanner scan = new Scanner(System.in);
+    ArrayList<Item> Inventory = player.getInventory();
 
-   @Override
-   public void enter() throws PlayerDeadException {
+    public void printRoomMessage() {
+        System.out.println("Um weiter zu kommen musst du deinen Gegner bezwingen");
+    }
 
-      GameInput in = new GameInput(new InputStreamReader(System.in));
-      Enemy enemy = new Enemy(); // Name übergeben von Enemy
-      Random random = new Random();
+    @Override
+    public void printWelcomeMessage() {
+        System.out.println("Du stehst einem mächtigen Gegner gegenüber.");
+    }
 
-      System.out.println("Du bist gefangen und kommst nur raus, wenn du den Gegner tötest, oder 'quit' drückst");
-      System.out.println("Attack Damage Player: " + Player.getPlayerInstance().getHealthState());
-      System.out.println("Attack Damage Enemy: " + enemy.getHealthState());
+    @Override
+    public void enter() throws PlayerDeadException {
 
-      while (in.gameState()) {
-         System.out.println("Drücke 1) um anzugreifen oder 2) um zu verteidigen.");
 
-         try {
-            fight(in, enemy, random);
-         } catch (InvalidInputException e) {
-            System.out.println(e.getMessage());
-         }
+        GameInput in = new GameInput(new InputStreamReader(System.in));
+        Enemy enemy = new Enemy();
+        Weapon equippedWeapon = player.getEquippedWeapon();
```

Comment:
Man muss hier nicht equippedWeapon übergebn, da du einfach in der Methode ds Weapon abfragen kannst